transport/http/handle.go

```diff
@@ -122,8 +115,25 @@ func encodeError(w http.ResponseWriter, r *http.Request, err error) {
 			Message: err.Error(),
 		}
 	}
+	codec := getCodec(r)
+	data, _ := codec.Marshal(se)
+	w.Header().Set(contentTypeHeader, contentType(codec.Name()))
 	w.WriteHeader(se.HTTPStatus())
-	encodeResponse(w, r, se)
+	_, _ = w.Write(data)
+}
+
+// getCodec get encoding.Codec via http.Request
+func getCodec(r *http.Request) encoding.Codec {
```

Comment:
codecForRequest(r *http.Request)

Reply:
Maybe `codecFromRequest` is better?

Reply:
https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#ReasonForError
https://pkg.go.dev/github.com/grpc-ecosystem/grpc-gateway/runtime#MarshalerForRequest

This method can be likes grpc gataway.

Reply:
Thanks